spec/models/manageiq/providers/ibm_power_vc/cloud_manager/refresher_spec.rb

```diff
@@ -2,6 +2,23 @@
   let(:zone) { EvmSpecHelper.create_guid_miq_server_zone.last }
   let(:ems) { FactoryBot.create(:ems_ibm_power_vc_with_vcr_authentication, :zone => zone) }
 
+  specs = YAML.load_file(File.join(__dir__, 'state.yml'))
+  specs['resources'].each do |key, _value|
```

Comment:
I'm good with this for now but I feel like we can improve this by converting to a hash, something like:
```ruby
resources_by_type = specs['resources'].group_by { |key, _val| key['type'] }
```

We could even go further and build a hash on type and name if it is unique so we would just have `let(:flavorid) { resources["openstack_compute_flavor_v2"]["flavor"]`

Reply:
Definetly. We have just created a simple test for a single Power VM.  Right now since we are just checking few attributes, I just hardcoded these. We will have to revisit it once we have have enhanced our test specs.